<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateVolumeDialog</class>
 <widget class="QDialog" name="CreateVolumeDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>warning!!</string>
  </property>
  <widget class="QPushButton" name="pbYes">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>This operation will lead to permanent destrunction of all present data in /dev/sdc1. Are you sure you want to continue?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
